dronegen/mac.go

```diff
@@ -521,7 +521,7 @@ func darwinBuildCommands(toolchainConfig toolchainConfig, artifactConfig darwinA
 			`cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport`,
 			`build.assets/build-fido2-macos.sh build`,
 			`export PKG_CONFIG_PATH="$(build.assets/build-fido2-macos.sh pkg_config_path)"`,
-			`make clean release OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes`,
+			`make clean release OS=$OS ARCH=$ARCH FIDO2=yes TOUCHID=yes LIBPCSCLITE=yes`,
```

Comment:
Is libpcsclite preinstalled on macOS?

Reply:
That's what I thought, but actually it's just not required for macOS or Windows to compile `piv-go`. I removed the tag from here and added the `!linux` build tag to `keys/yubikey.go`. 

Let me know if I'm missing any other important build steps, I haven't touched the new build flow much.